packages/communication-react/review/communication-react.api.md

```diff
@@ -1719,6 +1720,7 @@ export const SendBox: (props: SendBoxProps) => JSX.Element;
 // @public
 export interface SendBoxProps {
     disabled?: boolean;
+    hasFocusOnMount?: 'sendBoxTextField' | false;
```

Comment:
Should this just be `boolean`. In the case of this component, we wouldn't want to provide multiple focus points, so keep the API simple, and just name it to `autoFocus`, as we're simply forwarding the prop to the HTML element for the TextBox, right?

Reply:
so move the boolean comparison in the inputboxComponent out to the chat screen inside the composite?

Reply:
I'm happy for either boolean or `'sendBoxTextField' | false;`.
There is a send button inside the component so I'm happy to be non-opinionated and say more-than-one-component == use-the-enum.
But I'm also happy for us to be opinionated and say its a sendbox - we will want the user to be able to type something in the box before hitting send